processor/src/tests/crypto_ops.rs

```diff
@@ -0,0 +1,150 @@
+use super::{super::execute, build_inputs, compile};
+use super::{build_stack_state, Felt, FieldElement};
+use crate::Word;
+use rand_utils::rand_vector;
+use vm_core::{AdviceSet, ProgramInputs, StarkField};
+use winterfell::crypto::{hashers::Rp64_256, ElementHasher};
```

Comment:
Could we sue `core::hasher` module instead here?

Reply:
Yes. FYI the unit tests in the [processor's crypto_ops file](https://github.com/maticnetwork/miden/blob/next/processor/src/operations/crypto_ops.rs) are importing the hasher from winterfell as well. Maybe that should change too?

Reply:
Yes, let's do that!